lib/trento_web/views/v1/sap_system_view.ex

```diff
@@ -97,6 +97,21 @@ defmodule TrentoWeb.V1.SapSystemView do
 
   def render("application_instance_health_changed.json", %{health: health}), do: health
 
+  def render("instance_absent_at_changed.json", %{
```

Comment:
Let's put this function at the end, just after `def render("instance_deregistered.json", %{`